tests/plotting/_images/test_plotting.py

```diff
@@ -0,0 +1,35 @@
+from anndata import AnnData
+import scanpy as sc
+
+from tests.plotting.conftest import PlotTester, PlotTesterMeta
+import moscot.plotting as mpl
+
+sc.pl.set_rcParams_defaults()
```

Comment:
This should be done on `pytest_sessionstart`.

Reply:
Following [this](https://stackoverflow.com/questions/12586489/pytest-are-pytest-sessionstart-and-pytest-sessionfinish-valid-hooks) I created a fixture. 

Or did you have another `session_start` in mind?